hooks/useTimeSeriesMqlQuery/utils/useCreateTimeSeriesMqlQuery.ts

```diff
@@ -71,7 +71,8 @@ const useCreateTimeSeriesMqlQuery = ({
 
   const createTimeSeriesMqlQuery = ({ stateRetries }: DoCreateMqlQueryArgs) => {
     createMqlQueryMutation({
-      addTimeSeries: true,
+      addTimeSeries:
+        formState?.addTimeSeries === undefined ? true : formState.addTimeSeries,
```

Comment:
You could probably also do
```suggestion
        formState?.addTimeSeries !== false,
```